habitat/tasks/rearrange/sub_tasks/nav_to_obj_task.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, *args, config, dataset=None, **kwargs):
         cache_name += ",".join(self._config.FILTER_NAV_TO_TASKS)
         fname = data_path.split("/")[-1].split(".")[0]
         self.cache = CacheHelper(
-            "dyn_nav_start_pos", cache_name, {}, verbose=True, rel_dir=fname
+            "dyn_nav_start_pos", cache_name, {}, verbose=False, rel_dir=fname
```

Comment:
default verbosity on other tasks is False. 
TODO later : Need to make this a configuration flag